docs/articles/nunit/release-notes/Nunit4.0-MigrationGuide.md

```diff
@@ -157,6 +159,25 @@ global using FileAssert = NUnit.Framework.Legacy.FileAssert;
 Note that this doesn't mean you have to target .NET 6.0. This also works if targeting .NET Framework as it is purely
 done on the source code level.
 
+If you have to do this for multiple project in a repository, this is not handy.
+It seems somebody else thought of [that](https://github.com/dotnet/msbuild/issues/6745) and
+it is now possible to add global usings in a
+[`Directory.Build.props`](https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-by-directory?view=vs-2022#directorybuildprops-and-directorybuildtargets)
+file which will be used by all projects in a repository:
+
+```xml
+<ItemGroup>
+  <Using Include="NUnit.Framework.Legacy.ClassicAssert" Alias="Assert" />
+  <Using Include="NUnit.Framework.Legacy.CollectionAssert" Alias="CollectionAssert" />
+  <Using Include="NUnit.Framework.Legacy.StringAssert" Alias="StringAssert" />
+  <Using Include="NUnit.Framework.Legacy.DirectoryAssert" Alias="DirectoryAssert" />
+  <Using Include="NUnit.Framework.Legacy.FileAssert" Alias="FileAssert" />
+</ItemGroup>
+
+The build process now automatically creates a `${Project}.GlobalUsings.g.cs` file
+for each project with a contents similar to shown above.
+```
```

Comment:
```suggestion
```xml
<ItemGroup>
  <Using Include="NUnit.Framework.Legacy.ClassicAssert" Alias="Assert" />
  <Using Include="NUnit.Framework.Legacy.CollectionAssert" Alias="CollectionAssert" />
  <Using Include="NUnit.Framework.Legacy.StringAssert" Alias="StringAssert" />
  <Using Include="NUnit.Framework.Legacy.DirectoryAssert" Alias="DirectoryAssert" />
  <Using Include="NUnit.Framework.Legacy.FileAssert" Alias="FileAssert" />
</ItemGroup>
```

The build process now automatically creates a `${Project}.GlobalUsings.g.cs` file
for each project with a contents similar to shown above.
```

Reply:
Thanks, I already fixed that one when I looked at the preview.